client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -180,6 +189,15 @@ final case class Client(open: Service[Request, DisposableResponse], shutdown: Ta
   def expect[A](req: Task[Request])(implicit d: EntityDecoder[A]): Task[A] =
     req.flatMap(expect(_)(d))
 
+  /** Submits a request and returns the response status */
+  def status(req: Task[Request]): Task[Status] =
```

Comment:
I think they were sorted by argument type instead of by name, but that's silly.  I'll shuffle it around.